robottelo/constants.py

```diff
@@ -258,9 +258,14 @@
 DEFAULT_SUBSCRIPTION_NAME = (
     'Red Hat Enterprise Linux Server Entry Level, Self-support')
 
-LANGUAGES = ["en", "en_GB",
-             "fr", "gl",
-             "ja", "sv_SE"]
+LANGUAGES = [
+    "zh_TW",
+    "Japanese",
+    "Deutsch",
+    "it",
+    "ko",
+    "sv_SE",
+]
```

Comment:
And basically, that doesn't make sense as that issue can be resolved from selenium side in next releases